source/_integrations/boschshc.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: Bosch Smart Home Controller Integration
+description: Instructions on how to integrate Bosch Smart Home Controller into Home Assistant.
+ha_category:
+  - Binary Sensor
+ha_release: 0.109
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@tschamm'
+  - '@cabrust'
+ha_domain: boschshc
+---
+
+The `boschshc` integration allows you to integrate your [Bosch SHC](https://www.bosch-smarthome.com) into Home Assistant.
+
+There is currently support for the following device types within Home Assistant:
+
+- [Configuration](#configuration)
+  - [Binary Sensor](#binary-sensor)
+- [Client registration](#client-registration)
+
+## Configuration
+
+You will need the IP Address of your SHC and a registered client with a valid SSL certificate/key pair to use this module.
+
+To add `Bosch SHC` to your installation, go to **Configuration** >> **Integrations** in the UI, click the button with `+` sign and from the list of integrations select **Bosch Smart Home Controller**.
+
+Alternatively, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+boschshc:
+   ip_address: YOUR_SHC_IP
+   ssl_certificate: YOUR_SSL_CERT
+   ssl_key: YOUR_SSL_KEY
+   name: YOUR_SHC_NAME
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.